HW03 - Carpet Installation/CarpetInstallation.c

```diff
@@ -30,28 +30,25 @@ int main() {
     //Prints the charges section.
     printCharges(length, width, costSF, installPrice, discount, subtotal, total);
 
-    return(0);
+    return 0;
```

Comment:
Please be consistent with the code style of the rest of the file and either  maintain parenthesis on all returns or remove them all.